jsk_perception/node_scripts/tile_image.py

```diff
@@ -47,8 +48,10 @@ def unsubscribe(self):
     def _apply(self, *msgs):
         bridge = cv_bridge.CvBridge()
         imgs = []
-        for msg in msgs:
+        for msg, topic in zip(msgs, self.input_topics):
             img = bridge.imgmsg_to_cv2(msg, desired_encoding='bgr8')
+            if self.draw_topic_name:
```

Comment:
Maybe you don't need to set value to property but directly see the rosparam.
like
```python
if rospy.get_param('~draw_topic_name', False):
   ...
```